Packs/PANOSPolicyOptimizer/ReleaseNotes/1_1_27.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### PAN-OS Policy Optimizer (Beta)
+
+Added support for Panorama from version 10.1.10.
```

Comment:
do we mention the supported version in README.md? If so, it should be updated.

Reply:
Done